### PR TITLE
Prepare 0.3.1

### PR DESCRIPTION
**Pull request by moreati**
Major changes from changelog

* #874 Support for Ansible 5 (ansible-core 2.12)
* #774 Fix bootstrap failures on macOS 11.x and 12.x, involving Python 2.7 wrapper
* #834 Support for Ansible 3 and 4 (ansible-core 2.11)
* #869 Continuous Integration tests are now run with Tox
* #869 Continuous Integration tests now cover CentOS 6 & 8, Debian 9 & 11, Ubuntu 16.04 & 20.04
* #860 Add initial support for podman connection (w/o Ansible support yet)
* #873 `python -c ...` first stage no longer uses :py:mod:`platform`` to detect the macOS release
* #876 `python -c ...` first stage no longer contains tab characters, to reduce size
* #878 Continuous Integration tests now correctly perform comparisons of 2 digit versions
* #878 Kubectl connector fixed with Ansible 2.10 and above

### CODE REVIEW

**Comment by moreati** [comment]
David,
1. Do you still wish to require a review for merges to the stable branch (that releases are taken from)?
2. Would you be able to look at this over Christmas, or in the New Year? I'm thinking a first half January release.

**Comment by maxpain** [comment]
Any news?

File: mitogen/__init__.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 
 #: Library version as a tuple.
-__version__ = (0, 3, 0)
+__version__ = (0, 3, 1, 'dev0')
```
Comment (dw) [inline mitogen/__init__.py:38]: Just noting this is still `dev0`, needs change before release

**Review by dw — APPROVED** [review]
LGTM! Loving the test tag changes, looks very convenient. The only thing I spotted was the version number, but guessing you already know about it. Thanks & happy new year

**Comment by dw** [comment]
> Do you still wish to require a review for merges to the stable branch (that releases are taken from)?

Please "@dw" me if you think it's an important change, and allow a little time to respond (no more than a few days max), but don't let it stop you from merging changes you are confident in, or if you feel waiting is blocking you.

File: mitogen/__init__.py (hunk 1 above)
Comment (moreati) [reply mitogen/__init__.py:38]: thanks, plan is to change that together with setting a release date in the changelog.

**Comment by maxpain** [comment]
Thank you, guys!

**Comment by saibug** [comment]
Hi community, at this moment there is mitogen version that support ansible 2.12 ?

**Pull request merged**